### DISCUSSION TITLE
Taro 编译的字节小程序  能在飞书上运行么

### DISCUSSION THREAD

**Discussion by Ailian0206** [NervJS/taro#9997] [Q & A]
用taro 编译的小程序  在字节开发者工具上  和在飞书小程序上表现的不一样 请问字节小程序是不能在飞书上运行么

**Comment by honlyHuang** [comment]
可以直接编译成飞书小程序啦，飞书插件 https://github.com/NervJS/taro-plugin-platform-lark
npm 地址：https://www.npmjs.com/package/@tarojs/plugin-platform-lark

**Reply by huhaifan** [reply]
飞书插件地址404了，钉钉小程序的插件也修复下啊，选型的时候都调不通是把开发者逼回原生啊
